Sources/XMTPiOS/Client.swift

```diff
@@ -100,23 +110,48 @@ public final class Client: Sendable {
 	}
 
 	static func create(account: SigningKey, apiClient: ApiClient, options: ClientOptions? = nil) async throws -> Client {
-		let privateKeyBundleV1 = try await loadOrCreateKeys(for: account, apiClient: apiClient, options: options)
+		let (privateKeyBundleV1, source) = try await loadOrCreateKeys(for: account, apiClient: apiClient, options: options)
+
+		let v3Client: FfiXmtpClient?
+
+		if options?.enableAlphaMLS == true && options?.api.env == .local {
+			let dbURL = URL.documentsDirectory.appendingPathComponent("xmtp-\(account.address).db3")
```

Comment:
Nick suggested in my PR also adding the environment so we don't have database conflicts between envs

`val alias = "xmtp-${options!!.api.env}-${account.address.lowercase()}"`

Reply:
Good call, updating